front/lib/plans/usage/mau.ts

```diff
@@ -74,7 +74,7 @@ export async function reportMonthlyActiveUsers(
     );
   }
 
-  const activeUsers = await getActiveUsersInWorkspaceForPeriod({
+  const activeUsers = await countActiveUsersForPeriodInWorkspace({
```

Comment:
nit: would name activeUsersCount (very nit here I agree)

Reply:
I'll rename when moving those to the `WorkspaceResource`.